1-introduction/standard_lib_Task3.py

```diff
@@ -0,0 +1,9 @@
+from csv import DictReader
+from json import dump
+
+
+with open('cars.csv') as f:
+    reader = DictReader(f)
+    csv_dict = [row for row in reader]
```

Comment:
I'd make it simple like `csv_dict = list(reader)`

Reply:
That's a good point - I've updated this line - thank you!